### PR TITLE
HTTP routing and fault injection commands

### PR DESCRIPTION
**Pull request by waynz0r**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | yes
| API breaks?     | no
| Deprecations?   | no
| Related tickets | 
| License         | Apache 2.0

### What's in this PR?

- Adds http routing management commands
- Adds fault injection management commands

### CODE REVIEW

File: pkg/questionnaire/questionnaire.go [hunk 1]
```diff
@@ -17,48 +17,78 @@ package questionnaire
...
+			case reflect.Float32:
+				def = strconv.Itoa(int(elem.Interface().(float32)))
+			case reflect.Int:
+				def = strconv.Itoa(elem.Interface().(int))
 			case reflect.Int32:
 				def = strconv.Itoa(int(elem.Interface().(int32)))
+			case reflect.Int64:
+				def = strconv.Itoa(int(elem.Interface().(int64)))
 			case reflect.String:
 				def = elem.Interface().(string)
```
Comment (pepov) [inline pkg/questionnaire/questionnaire.go]: have you considered using spf13/cast for simplicity here?

File: internal/cli/cmd/routing/fi/delete.go [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+		return errors.WrapIf(err, "could not get initialized graphql client")
+	}
+	defer client.Close()
+
+	service, err := client.GetService(options.serviceName.Namespace, options.serviceName.Name)
+	if err != nil {
+		return errors.WrapIf(err, "could not get service")
+	}
+
+	r, err := client.GetService(service.Namespace, service.Name)
```
Comment (baluchicken) [inline internal/cli/cmd/routing/fi/delete.go]: Why we need to get the service twice?

File: internal/cli/cmd/routing/fi/get.go [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+		return errors.WrapIf(err, "could not get initialized graphql client")
+	}
+	defer client.Close()
+
+	service, err := client.GetService(options.serviceName.Namespace, options.serviceName.Name)
+	if err != nil {
+		return errors.WrapIf(err, "could not get service")
+	}
+
+	r, err := client.GetService(service.Namespace, service.Name)
```
Comment (baluchicken) [inline internal/cli/cmd/routing/fi/get.go]: same here.

Comment (waynz0r) [reply internal/cli/cmd/routing/fi/get.go]: fixed, thanks!

**Pull request merged**